security/ee-security-basic/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2015 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+  -->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.thorntail.examples</groupId>
+    <artifactId>examples-jaxrs</artifactId>
```

Comment:
This is probably what the build fails on -- should be `examples-security`.

Reply:
Fixed too.

Is it possible to get read access to CI server? I can't access the build info when anonymous and when I log in I can access nothing at all.

Reply:
I tried to figure out why build logs are no longer accessible to anonymous users, but couldn't find anything :-( I'll take a look again. In the meantime, I tried to enable all authenticated users to have read permissions. Can you try now?

Reply:
Now it behaves the same logged in or not. Trying to access e.g. https://ci.wildfly-swarm.io/view/Examples/job/thorntail-examples-pull-request-linux/ or a particular build gives 404.